docs/BUILD

```diff
@@ -16,7 +16,9 @@ filegroup(
             "root/**/*.yaml",
         ],
         exclude = [
-            "root/**/envoy-dynamic*.yaml",
+            "root/**/envoy-dynamic-cds-demo.yaml",
```

Comment:
Not sure I understand: these expand the exclude list and the files won't be present in the produced file list, while the PR comment says that you wanted to validate them?

Reply:
the original glob excluded all of the `envoy-dynamic*.yaml` altho one that matched was bootstrap and should be validated

Reply:
ahh, I see now.